src/Features/LanguageServer/Protocol/Features/CodeFixes/CodeFixService.cs

```diff
@@ -220,6 +256,21 @@ internal partial class CodeFixService : ICodeFixService
                 }
             }
 
+            if (!copilotDiagnostics.IsEmpty)
+            {
+                var text = await document.GetValueTextAsync(cancellationToken).ConfigureAwait(false);
```

Comment:
what is happening here?  this all needs doc'ing.

Reply:
We are following the exact same pattern as the prior diagnostics buckets in this method.